test-runner/README.md

```diff
@@ -26,4 +41,25 @@ cd mesos-spark-integration-tests
 sbt "mit <spark_home> mesos://<mesos-master-ip>:5050"
 ```
 
-*We use spark_submit.sh from <SPARK_HOME> to submit jobs. So please make sure you have a Spark binary distribution downloaded and unzipped*
+*We use spark_submit.sh from <SPARK_HOME> to submit jobs. So please
+ make sure you have a Spark binary distribution downloaded and
+ unzipped*
+
+### DCOS cluster
+
+The DCOS test runner uploads the Spark test JAR to S3 so the cluster
+can access it.  So the following Env Vars are required:
+
+```sh
+AWS_ACCESS_KEY
+AWS_SECRET_KEY
+S3_BUCKET
+S3_PREFIX
```

Comment:
These are now part of `dcos-application.conf`, as the [code](https://github.com/typesafehub/mesos-spark-integration-tests/pull/15/files#diff-7dbbcb9cf57c6f825618092f83338778R41) says.